WordPress/src/main/java/org/wordpress/android/ui/reader/repository/usecases/FollowInterestTagsUseCase.kt

```diff
@@ -0,0 +1,54 @@
+package org.wordpress.android.ui.reader.repository.usecases
+
+import org.greenrobot.eventbus.Subscribe
+import org.greenrobot.eventbus.ThreadMode
+import org.wordpress.android.models.ReaderTag
+import org.wordpress.android.ui.reader.ReaderEvents.FollowedTagsChanged
+import org.wordpress.android.ui.reader.actions.ReaderTagActions
+import org.wordpress.android.ui.reader.repository.ReaderRepositoryCommunication
+import org.wordpress.android.ui.reader.repository.ReaderRepositoryCommunication.Error.NetworkUnavailable
+import org.wordpress.android.ui.reader.repository.ReaderRepositoryCommunication.Error.RemoteRequestFailure
+import org.wordpress.android.ui.reader.repository.ReaderRepositoryCommunication.Success
+import org.wordpress.android.util.EventBusWrapper
+import org.wordpress.android.util.NetworkUtilsWrapper
+import javax.inject.Inject
+import kotlin.coroutines.Continuation
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
+
+class FollowInterestTagsUseCase @Inject constructor(
```

Comment:
We might want to rename it as `FollowTagsUseCase` so that we can reuse it when following a single tag as well. Do we want to structure it more like `PostLikeActionUseCase` ?